entries/c/commercial.metrobankonline.co.uk.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Metro Bank Commercial": {
+    "domain": "commercial.metrobankonline.co.uk",
+    "img": "metrobankonline.co.uk.svg",
+    "tfa": [
+      "custom-software"
```

Comment:
What about the "security device" (i.e. `custom-hardware`)?

See https://www.metrobankonline.co.uk/help-and-support/what-is-sca/#item15017
> You’ll need to use your security device or your Metro Bank Authenticator App to log in, ...